crates/store/re_protos/src/v1alpha1/rerun.manifest_registry.v1alpha1.ext.rs

```diff
@@ -266,13 +255,51 @@ pub struct FuzzyComponentDescriptor {
 
 #[derive(Debug, Clone)]
 pub struct QueryLatestAt {
-    pub index: String,
-    pub at: i64,
+    /// Index name (timeline) to query.
+    ///
+    /// Use `None` for static only data.
+    pub index: Option<String>,
+    /// The timestamp to query at.
+    ///
+    /// Use `TimeInt::STATIC` to query for static only data.
+    pub at: TimeInt,
+
     pub fuzzy_descriptors: Vec<String>,
     // TODO(cmc): I shall bring that back into a more structured form later.
     // pub fuzzy_descriptors: Vec<FuzzyComponentDescriptor>,
 }
 
+impl QueryLatestAt {
+    pub fn new_static() -> Self {
+        Self {
+            index: None,
+            at: TimeInt::STATIC,
+            fuzzy_descriptors: Vec::new(),
+        }
+    }
+
+    pub fn is_static(&self) -> bool {
+        self.index.is_none() && self.at.is_static()
```

Comment:
I'm not a big fan of the fact that the value of `self.at` even matters at all here. A lack of index should mean static, always, period.
(I wouldn't consider this to be a nit: these kinds of multi-field invariants open so many classes of bugs...)